### DISCUSSION TITLE
OneButtonPrompt

### DISCUSSION THREAD

**Discussion by almfahd41** [seruva19/kubin#79] [Ideas]
hello 

can you add this https://github.com/AIrjen/OneButtonPrompt/ ?

**Comment by seruva19** [comment]
Hello.
Unfortunately, extensions for Automatic1111 are not compatible with this app and OneButtonPrompt seems pretty complex extension, so I can't just port it without serious efforts. I am planning to make similar extension in future, so I will borrow some ideas (and probably csv files too 😅) from OneButtonPrompt.

**Reply by almfahd41** [reply]
yes i was asking for similar extension ,

in the Mixing tab I make both  the prompt empty and i get interesting results

**Reply by adrianpuiu** [reply]
so I will borrow some ideas (and probably csv files too sweat_smile) from OneButtonPrompt.

this reminds me of : "I would to thank stackoverflow for my license degree!" :))))))
